### PR TITLE
Mainly added debug

### PR DESCRIPTION
**Pull request by the-horseman**

### CODE REVIEW

File: cpp.json [hunk 1]
```diff
@@ -61,10 +104,10 @@
 	"online judge": {
 		"prefix": "onj",
 		"body": [
-			"#ifndef ONLINE_JUDGE",
-			"\tfreopen(\"${1:input.txt}\",\"r\",stdin);",
-			"\tfreopen(\"${2:output.txt}\",\"w\",stdout);",
-			"#endif"
+			"\n#ifndef ONLINE_JUDGE",
+    		"\tfreopen(\"in.txt\", \"r\", stdin);",
```
Comment (prabhatmalhan) [inline cpp.json]: Let it be input

File: cpp.json [hunk 2]
```diff
@@ -61,10 +104,10 @@
...
+    		"\tfreopen(\"out.txt\", \"w\", stdout);",
```
Comment (prabhatmalhan) [inline cpp.json]: Let it be output.txt

File: cpp.json (hunk 1 above)
Comment (prabhatmalhan) [reply cpp.json]: input.txt*

**Review by prabhatmalhan — CHANGES_REQUESTED** [review]

**Comment by the-horseman** [comment]
krdiye changes

**Pull request closed without merging**